src/features/harborView/forms/berthForm/BerthForm.tsx

```diff
@@ -49,9 +68,9 @@ const transformValues = (values: any): Berth => {
   return {
     ...values,
     number: parseInt(number),
-    width: parseFloat(width),
-    length: parseFloat(length),
-    depth: parseFloat(depth),
+    width: parseFloat(String(width).replace(',', '.')),
+    length: parseFloat(String(length).replace(',', '.')),
+    depth: parseFloat(String(depth).replace(',', '.')),
```

Comment:
Does it make sense to write a utility function for this conversion?

Reply:
I vote for utility function 👍